.pipelines/cni/singletenancy/linux-cniv2-template.yaml

```diff
@@ -0,0 +1,179 @@
+          - ${{ if contains(parameters.clusterType, 'swift') }}:
+            - task: AzureCLI@1
+              inputs:
+                azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+                scriptLocation: "inlineScript"
+                scriptType: "bash"
+                addSpnToEnvironment: true
+                inlineScript: |
+                  echo "Start Integration Tests on Swift Cluster"
+                  make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(commitID)
+                  kubectl cluster-info
+                  kubectl get po -owide -A
+                  sudo -E env "PATH=$PATH" make test-integration CNS_VERSION=$(cnsVersion) CNI_DROPGZ_VERSION=$(dropgzVersion) INSTALL_CNS=true INSTALL_AZURE_VNET=true TEST_DROPGZ=${{ parameters.testDropgz }}
+              name: "swifte2e"
+              displayName: "Swift Integration"
+      - template: ../../npm/npm-cni-integration-test.yaml
+        parameters:
+          clusterName: ${{ parameters.clusterName }}-$(commitID)
+          os: ${{ parameters.os }}
+          sub: $(TEST_SUB_SERVICE_CONNECTION)
+          tag: $(npmVersion)
+          dependsOn: integration
+      - job: deploy_pods
+        displayName: "Scale Test"
+        dependsOn: npm_k8se2e
+        steps:
+          - template: ../load-test-templates/pod-deployment-template.yaml
+            parameters:
+              clusterName: ${{ parameters.clusterName }}-$(commitID)
+              scaleup: ${LINUX_SCALEUP}
+              os: ${{ parameters.os }}
+              iterations: ${LINUX_ITERATIONS}
+          - template: ../load-test-templates/validate-state-template.yaml
+            parameters:
+              clusterName: ${{ parameters.clusterName }}-$(commitID)
+              os: ${{ parameters.os }}
+              cni: cniv2
+      - job: restart_nodes
+        displayName: "Restart Test"
+        dependsOn: deploy_pods
+        steps:
+          - template: ../load-test-templates/restart-node-template.yaml
+            parameters:
+              clusterName: ${{ parameters.clusterName }}-$(commitID)
+          - template: ../load-test-templates/validate-state-template.yaml
+            parameters:
+              clusterName: ${{ parameters.clusterName }}-$(commitID)
+              os: ${{ parameters.os }}
+              cni: cniv2
+              restartCase: "true"
+      - job: recover
+        displayName: "Recover Resources"
+        dependsOn: restart_nodes
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                echo "Delete load-test Namespace"
+                make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(commitID)
+                kubectl delete ns load-test
```

Comment:
Why do we want to delete the namespace ? Isn't it good that CNS restart is tested with certain number of pods.

Reply:
The bug I found while working this was sometime nodes would reach max pods inconsistently with one another. Leaving one node, or more, unable to schedule the privileged pod. Should we do a 90% or 95% scale on the restart case to prevent this instead?

Reply:
Yeah that would work.
The intension is to have more than `batchsize`(i.e. 16) pods on the node when the cns restarts.

Reply:
As an extension, we do we need to remove the pods before k8s e2e tests.